src/modules.c

```diff
@@ -543,7 +543,7 @@ Function global_table[] = {
   /* 252 - 255 */
   (Function) egg_snprintf,
   (Function) egg_vsnprintf,
-  (Function) egg_memset,
+  (Function) 0,                   /* was egg_memset -- use memset() or egg_bzero() instead */
```

Comment:
could we export memset if it has the same signature at this point to avoid breaking modules?

Reply:
yes, good idea.